std_msgs/Header header
scene_graph/DetectedObject[] objects